exe_5.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+def add_vector_to_matrix(vector:np, matrix:np)->np:
+    """
+    :param vector: Vector to add to all columns in the matrix.
+    :param matrix: Some Matrix.
+    :return: matrix with the addition of a vector to each row of the matrix.
+    """
+    return np.array([matrix[i, :] + vector for i in range(4)])

```

Comment:
The beauty of numpy is that in most cases for loops are not needed (and are usually less efficient), try using broadcasting.
https://numpy.org/doc/stable/user/basics.broadcasting.html